src/components/AboutTab/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Text, View } from "react-native";
+
+import convertToTitleCase from "../../_utils/convertToTitleCase";
+import type { Pokemon } from "../../types/interfaces/Pokemon";
+
+interface AboutTabProps {
+    viewedPokemon: Pokemon;
+}
+
+export default function AboutTab({ viewedPokemon }: AboutTabProps) {
+    return (
+        <View className="font-chakra">
+            <Text className="pb-3 text-2xl font-chakra-bold">
+                Pokemon Details
+            </Text>
+            <View className="flex flex-row gap-10 ">
+                <View className="flex">
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Name
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra ">
+                        Height
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Weight
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Base Experience
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Abilities
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Types
+                    </Text>
```

Comment:
Is there a way we can extract common class names under one class name so we just use that instead of repeating the lengthy CSS every time?